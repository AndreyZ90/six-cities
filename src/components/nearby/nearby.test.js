import React from 'react';
import TestRenderer from 'react-test-renderer';

import Nearby from '@/components/nearby/nearby';

const childrenNode = <div className="some-children-node">Nearby children node</div>;

describe(`Nearby component snapshot`, () => {
  test(`Should correctly render Nearby component`, () => {
    const tree = TestRenderer.create(
        <Nearby>
          {childrenNode}
        </Nearby>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
