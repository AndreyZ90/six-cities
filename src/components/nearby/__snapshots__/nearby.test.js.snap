// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nearby component snapshot Should correctly render Nearby component 1`] = `
<section
  className="near-places places"
>
  <h2
    className="near-places__title"
  >
    Other places in the neighbourhood
  </h2>
  <div
    className="some-children-node"
  >
    Nearby children node
  </div>
</section>
`;
